app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -227,8 +227,6 @@ def user_agent
 
       def search_action_for_dashboard
         case params[:controller]
-        when "hyrax/my/works"
-          hyrax.my_works_path
```

Comment:
Removing this case since it duplicates the default case. Mostly this was done to avoid violating Rubocop's `Metrics/CyclomaticComplexity` and `Metrics/MethodLength` rules.